docs/source/libraries/keras.rst

```diff
@@ -0,0 +1,40 @@
+.. _library-keras:
+
+Keras
+=====
+
+Keras_ is "a high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano". 
+
+Currently ELI5 supports :func:`eli5.explain_prediction` for Keras image classifiers.
+
+.. _Keras: https://keras.io/
+
+:func:`eli5.explain_prediction` explains image classifications through `Grad-CAM <https://arxiv.org/pdf/1610.02391.pdf>`_. The :class:`eli5.base.Explanation` object returned has a ``.image`` attribute that represents the  image that is inputted into the model, and a ``.heatmap`` attribute that is a grayscale "localization map" roughly indicating regions of importance in the image for the predicted class.
+
+Important arguments for ``Model`` and ``Sequential``:
+
+* ``doc`` is an image as a tensor that can be inputted to the model.
```

Comment:
Do we have any more specific requirements? Must it have batch dimension? Specific layout (e.g. BHWC)? Do we support anything besides numpy arrays? Docstring below in ``explain_prediction_keras`` is much more specific, I think it makes sense to be as specific here.

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/64a31a190f50d9446ae029fa4e24f88983de9f22